config/initializers/storybook.rb

```diff
@@ -0,0 +1,14 @@
+# Adjust headers to allow running Storybook in development.
+# Uses iframes and doesn't play nicely with CORS checks
+
+if Rails.env.development?
+  module PermissiveCORSHeaders
+    def self.before(response)
+      response.headers["Access-Control-Allow-Origin"] = "*"
+      response.headers["Access-Control-Allow-Methods"] = "GET"
```

Comment:
I think after the 5.2 upgrade is merged we could even replace this with something slightly nicer, as 5.2 introduces a new built-in DSL for modifying response headers :tada: